test/e2e/network/dns_common.go

```diff
@@ -430,7 +429,7 @@ func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookup
 	fileNames = append(fileNames, podARecByTCPFileName)
 
 	if len(ptrLookupIP) > 0 {
-		ptrLookup, err := dnsutil.ReverseAddr(ptrLookupIP)
+		ptrLookup, err := reverseAddr(ptrLookupIP)
```

Comment:
can we use https://godoc.org/net#LookupAddr instead?

Reply:
edit: oh, nevermind, misread goLookupPTR... it doesn't return the result of reverseaddr, that's an intermediate internal result

Reply:
`LookupAddr` performs the DNS resolution, but this has to be done from the `testpod`.
`reverseAddr` return the format of the DNS record, that is used to create the probes in L438 and L439